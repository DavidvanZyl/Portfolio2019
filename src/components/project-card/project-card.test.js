import React from "react"
import ProjectCard from "./project-card"

const testItem = {
  node: {
    uid: "test",
    data: {
      item_image: {
        url: "test.jpg",
        alt: "test image",
      },
      title: {
        text: "test",
      },
    },
  },
}

describe("BlogEntry", () => {
  test("renders correctly", () => {
    const wrapper = shallow(<ProjectCard item={testItem} />)
    expect(wrapper).toMatchSnapshot()
  })
})
