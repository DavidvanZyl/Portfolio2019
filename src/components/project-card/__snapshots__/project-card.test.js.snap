// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogEntry renders correctly 1`] = `
<ForwardRef
  className="project"
  to="/project/test"
>
  <div
    className="filtered"
  >
    <img
      alt="test image"
      src="test.jpg"
    />
  </div>
  <p
    className="project__title"
  >
    test
  </p>
</ForwardRef>
`;
